File: app/src/main/java/deakin/gopher/guardian/view/caretaker/notifications/FallAlertActivity.java
package deakin.gopher.guardian.view.caretaker.notifications;

import android.content.Intent;
import android.os.Bundle;
import android.widget.ImageButton;
import android.widget.ImageView;
import androidx.core.view.GravityCompat;
import androidx.drawerlayout.widget.DrawerLayout;
import com.google.android.material.navigation.NavigationView;
import deakin.gopher.guardian.R;
import deakin.gopher.guardian.view.caretaker.notifications.confirmincident.ConfirmIncidentActivity;
import deakin.gopher.guardian.view.caretaker.notifications.falsealarm.FalseAlertConfirmedActivity;
import deakin.gopher.guardian.view.general.BaseActivity;

public class FallAlertActivity extends BaseActivity {

  ImageView fallAlertMenuButton;

  @Override
  protected void onCreate(final Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_fall_alert);

    final NavigationView navigationView = findViewById(R.id.nav_view);
    fallAlertMenuButton = findViewById(R.id.menuButton);
    final DrawerLayout drawerLayout = findViewById(R.id.drawer_layout);
    navigationView.setItemIconTintList(null);

    fallAlertMenuButton.setOnClickListener(
        v -> {
          drawerLayout.openDrawer(GravityCompat.START);
        });

    final ImageButton confirmIncidentButton = findViewById(R.id.confirmIncidentButton);
    final ImageButton falseAlarmButton = findViewById(R.id.falseAlarmButton);

    confirmIncidentButton.setOnClickListener(
        v -> {
          final Intent medicalDiagnosticsActivityIntent =
              new Intent(FallAlertActivity.this, ConfirmIncidentActivity.class);
          startActivity(medicalDiagnosticsActivityIntent);
        });

    falseAlarmButton.setOnClickListener(
        v -> {
          final Intent medicalDiagnosticsActivityIntent =
              new Intent(FallAlertActivity.this, FalseAlertConfirmedActivity.class);
          startActivity(medicalDiagnosticsActivityIntent);
        });
  }
}

File: app/src/main/java/deakin/gopher/guardian/view/caretaker/notifications/FallAlertActivity.kt
package deakin.gopher.guardian.view.caretaker.notifications

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.ImageButton
import android.widget.ImageView
import androidx.core.view.GravityCompat
import androidx.drawerlayout.widget.DrawerLayout
import com.google.android.material.navigation.NavigationView
import deakin.gopher.guardian.R
import deakin.gopher.guardian.view.caretaker.notifications.confirmincident.ConfirmIncidentActivity
import deakin.gopher.guardian.view.caretaker.notifications.falsealarm.FalseAlertConfirmedActivity
import deakin.gopher.guardian.view.general.BaseActivity

class FallAlertActivity : BaseActivity() {
    var fallAlertMenuButton: ImageView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_fall_alert)

        val navigationView = findViewById<NavigationView>(R.id.nav_view)
        fallAlertMenuButton = findViewById(R.id.menuButton)
        val drawerLayout = findViewById<DrawerLayout>(R.id.drawer_layout)
        navigationView.itemIconTintList = null

        fallAlertMenuButton?.let { button ->
            button.setOnClickListener {
                drawerLayout.openDrawer(GravityCompat.START)
            }
        }

        val confirmIncidentButton = findViewById<ImageButton>(R.id.confirmIncidentButton)
        val falseAlarmButton = findViewById<ImageButton>(R.id.falseAlarmButton)

        confirmIncidentButton.setOnClickListener { v: View? ->
            val medicalDiagnosticsActivityIntent =
                Intent(this@FallAlertActivity, ConfirmIncidentActivity::class.java)
            startActivity(medicalDiagnosticsActivityIntent)
        }

        falseAlarmButton.setOnClickListener { v: View? ->
            val medicalDiagnosticsActivityIntent =
                Intent(this@FallAlertActivity, FalseAlertConfirmedActivity::class.java)
            startActivity(medicalDiagnosticsActivityIntent)
        }
    }
}
